solr/bin/solr

```diff
@@ -2089,6 +2089,14 @@ else
   SECURITY_MANAGER_OPTS=()
 fi
 
+# Enable ADMIN UI by default, and give the option for users to disable it when.
```

Comment:
[-1] The way I read this, the Admin UI will be disabled whenever SOLR_ADMIN_UI_ENABLED is empty, unset, etc.  And I don't see that value set anywhere, except in commented-out lines in solr.in.sh/solr.in.cmd.

So by default the Admin UI is disabled, which doesn't seem like your intention?  Or am I missing something?

Reply:
it's fixed. Psyched even myself out. I have also changed the names.